subiquity/models/filesystem.py

```diff
@@ -150,16 +223,49 @@ def from_info(self, info):
         d.model = info.model
         return d
 
+    def info_for_display(self):
+        bus = self._info.raw.get('ID_BUS', None)
```

Comment:
Generally shouldn't the items we fetch from _info.raw be attributes on the info object?  That allows it to be mocked or replaced and if where we fetch the ID_BUS or other values from won't brake here if that changes.

Reply:
Yes probably, I just moved this code though, can we clean it up later? :)

Reply:
Surely, for items deferred, we either open a bug or add a card to our tech debt backlog so we don't forget.